src/dpr/components/report-list/defaultTokenProvider.ts

```diff
@@ -0,0 +1,9 @@
+import { NextFunction, Request, Response } from 'express'
+
+export default (request: Request, response: Response, next: NextFunction): string => {
+  if (response.locals.user && response.locals.user.token) {
```

Comment:
This is more of a comment for the fake-pds-service-ui project.
Since this method is now here and the project is using the frontend library can we remove the duplication from there so that it calls the library method instead?

Reply:
Yes indeed!